tests/e2e/tests/e2e_util/util.py

```diff
@@ -104,9 +104,7 @@ def setup_network(h: harness.Harness, instance_id: str):
     h.exec(instance_id, ["/snap/k8s/current/k8s/network-requirements.sh"])
 
     LOG.info("Waiting for network to be enabled...")
-    stubbornly(retries=15, delay_s=5).until(
-        lambda p: "enabled" in p.stdout.decode()
-    ).exec(["k8s", "enable", "network"], h, instance_id)
```

Comment:
Stop relying on the output of `k8s enable ...` -- just enable it and trust the return codes